target-bin/complete-resize.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+DEBIAN_FRONTEND=noninteractive apt-get --no-install-recommends -y install expect-dev parted
+swapoff -a
+{ cat <<\EOF; sleep 1; while pidof parted; do sleep 1; done; } | unbuffer -p parted /dev/vda
+rm 2
```

Comment:
is this swap?  a bit worried it might break some builds that were on the edge with memory.  are you thinking swap is generally not needed?